### PR TITLE
Catch rare error with allele fraction calculation

### PR DESCRIPTION
**Pull request by achevali**

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/MinorAlleleFractionCache.java [hunk 1]
```diff
@@ -50,8 +51,11 @@ private static double estimateMinorAlleleFraction(final int a, final int r, fina
                     r * (logf + logBias) + a * logOneMinusf - (a + r) * logRefMinorDenominatorTerm;
             return GATKProtectedMathUtils.logSumExp(altMinorLogLikelihood, refMinorLogLikelihood);
         };
-
-        return OptimizationUtils.argmax(objective, AlleleFractionState.MIN_MINOR_FRACTION, AlleleFractionState.MAX_MINOR_FRACTION, initialEstimate);
+        try {
+            return OptimizationUtils.argmax(objective, AlleleFractionState.MIN_MINOR_FRACTION, AlleleFractionState.MAX_MINOR_FRACTION, initialEstimate);
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/MinorAlleleFractionCache.java]: Space between `catch` and the `{`.

File: src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/MinorAlleleFractionCache.java [hunk 2]
```diff
@@ -50,8 +51,11 @@ private static double estimateMinorAlleleFraction(final int a, final int r, fina
...
+        }catch (final TooManyArgumentsException ex){
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/MinorAlleleFractionCache.java]: It's a `TooManyEvaluationsException` from package `org.apache.commons.math3.exception`.

**Comment by davidbenjamin** [comment]
Back to @achevali.

**Comment by achevali** [comment]
Back to @davidbenjamin

**Comment by davidbenjamin** [comment]
@achevali Good to go.

**Pull request merged**